packages/components/src/product-icon/index.stories.jsx

```diff
@@ -5,7 +5,9 @@ import React from 'react';
 
 import './index.stories.scss';
 import ProductIcon from '.';
-import { supportedSlugs } from './config';
+import { iconToProductSlugMap } from './config';
+
+const supportedSlugs = [ ...Object.values( iconToProductSlugMap ) ].flat();
```

Comment:
`const supportedSlugs = Object.values( iconToProductSlugMap ).flat();` would be equivalent, right? (not sure if I'm missing something about how destructuring works here :))

Reply:
Oops, yup it would be, mindlessly copied this code from `config.js`, but I'll change it 👍🏼